internal/backoff/backoff_test.go

```diff
@@ -60,13 +60,13 @@ func TestLogBackoff(t *testing.T) {
 				WithSeed(0),
 			),
 			exp: []exp{
-				{gte: 0, lte: time.Second},
-				{gte: 0, lte: 2 * time.Second},
-				{gte: 0, lte: 4 * time.Second},
-				{gte: 0, lte: 8 * time.Second},
-				{gte: 0, lte: 8 * time.Second},
-				{gte: 0, lte: 8 * time.Second},
-				{gte: 0, lte: 8 * time.Second},
+				{eq: 0, gte: 0, lte: time.Second},
```

Comment:
is a linter required for tests too?
this code is too detailed for tests

Reply:
yes, it is.
exhaustruct linter checks all files, including tests.

Reply:
could you disable this linter for tests?

Reply:
I'll try if possible.
Now, I don't know how to switch off the linter for tests.